docs/staff/coaches/gamification.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 3
+---
+
+# Gamification
+
+|                       |                          |
+|-----------------------|--------------------------|
+| **Optional**          | Yes                      |
+| **Default**           | FULL                     |
+| **Feature Toggle**    | Gamification             |
+| **Can be enabled by** | Location Administrator   |
+| **Settings**          | Off, FULL, LEVEL_XP_ONLY |
+|                       |                          |
+
+### Description
+We have a couple of gamification features to inspire users to engage more with the content.
+
+## Level & XP
+
+![Level and XP bar](/img/staff/coaches/gamification/xp-bar.png)
+
+Users can collect experience points(XP) from different [actions](#wich-actions-yield-xp). 
+Once the user has reached enough XP for a level up, a level up screen will be shown and the user will have gained an extra level.
+After this the amount of XP needed for a level up is increased.
+
+![Level up screen](/img/staff/coaches/gamification/level-up.png)
+
+### Wich actions yield XP
+
+We calculate the amount of XP internally, we do not specify the exact amount of XP here, only a representation.
+
+| Action                            | XP yield |
+|-----------------------------------|----------|
+| Reviews                           | Low      |
+| Theory exercise completed         | Low      |
+| Practice exercise completed       | Medium   |
+| Self diagnostic completed         | Medium   |
+| Project completed                 | High     |
+
+**Good to know:**
+- The XP yield of theory and practice exercises changes with their difficulty. A higher difficulty and time indication will result in more XP.
+- If an exercise has been reopened or restarted, the same amount of XP that the exercise yielded, will be removed from the students profile.
+
+## Leaderboard
+
+The leaderboard shows the current students with the highest level and XP.
+
+![Leaderboard](/img/staff/coaches/gamification/leaderboard.png)
```

Comment:
Dit plaatje heeft rode profile pictures, misschien nog wel een goede om dat te vervangen. Kon alleen niet 123 een service vinden die plaatjes kon geven.

Reply:
In eerste instantie niet super erg lijkt me